rclcpp/test/test_client.cpp

```diff
@@ -75,7 +75,8 @@ TEST_F(TestClient, construction_and_destruction) {
   }
 
   {
-    ASSERT_THROW({
+    ASSERT_THROW(
+    {
```

Comment:
It looks weird that this is not indented. I guess you ran the linter on this patch and only this version passed?

Reply:
As it turns out, this is how I have to leave it. I tried intend_paren_open_brace, but this changed too much other stuff. As it turns out, this will be fixed by the other ament_uncrustify PR, but will involve some breaking PRs.